public/500.html

```diff
@@ -1,43 +0,0 @@
-<!DOCTYPE html>
-<html>
-<head>
-  <title>We're sorry, but something went wrong (500)</title>
-  <style type="text/css">
-    body {
-      background-color: #fff;
-      color: #666;
-      font-family: arial, sans-serif;
-
-    }
-    div.dialog {
-      width: 600px;
-      margin: auto;
-    }
-    @media only screen
-      and (min-width: 320px)
-      and (max-width: 568px) {
-        div.dialog, div.dialog img {
-          width: 95%;
-        }
-    }
-    div.dialog h1 {
-      font-size: 1.8em;
-    }
-  </style>
-</head>
-
-<body>
-  <!-- This file lives in public/500.html -->
-  <div class="dialog">
-    <a href="/" ><img src="/500.jpg" /></a>
```

Comment:
We should keep this file because Nginx uses it in case the app is completely down and can't render pages.
https://github.com/openfoodfoundation/ofn-install/blob/f7bb5362587f066bc798660a48371cd88ad6af57/inventory/group_vars/all.yml#L375

It would be even better to create a new page for that and manage it in ofn-install independent of the app...

Reply:
Oh nice catch!  🙏 

Maybe I let the file `500.html` here for this PR and open an issue onto `ofn-install` repo?

Reply:
https://github.com/openfoodfoundation/ofn-install/issues/824